index.php

```diff
@@ -9,14 +9,14 @@
 $change='';
 $percentChange='';
 $theTime='';
+$theDate='';
```

Comment:
Hmm yet another variable in the long list of variables.  We need to  look into this at a future date, but I will allow this for now